youths/models.py

```diff
@@ -22,6 +22,7 @@ class YouthProfile(models.Model):
     profile = models.OneToOneField(
         Profile, related_name="youth_profile", on_delete=models.CASCADE
     )
+    birth_date = models.DateField()
```

Comment:
is it obligatory?

Reply:
Yes, at least in my opinion. Without the birth date we cannot know how old the youth is. We have to know the age in order to validate certain things and contact the guardian for confirmation, if necessary.